tor-button@shatur/files/tor-button@shatur/applet.js

```diff
@@ -94,10 +105,10 @@ MyApplet.prototype = {
     
     on_applet_clicked: function() {
         if (torAppletCheck.substr(5,3) == "OFF") { // Check information from file without 'true,' and '\n '
-                                                   GLib.spawn_command_line_async(this.location); // Run tor from location settings        
+                                                   GLib.spawn_command_line_async(gksu+this.location); // Run tor from location settings        
```

Comment:
I don't think its a good idea to encourage running applications as root in the userland.